cscs-checks/prgenv/cuda/cuda_samples.py

```diff
@@ -51,14 +50,13 @@ def __init__(self):
         self.build_system = 'Make'
         self.build_system.options = [f'SMS="{self.nvidia_sm}"',
                                      f'CUDA_PATH=$CUDA_HOME']
-        self.prebuild_cmds = [f'git checkout v11.0',
-                              f'cd Samples/{self.test_name}']
-        self.executable = f'Samples/{self.test_name}/{self.test_name}'
+        self.prebuild_cmds = [f'git checkout v11.0', f'cd Samples/{sample}']
+        self.executable = f'Samples/{sample}/{sample}'
         self.sanity_patterns = sn.assert_found(
-            output_patterns[self.test_name], self.stdout
+            output_patterns[sample], self.stdout
         )
         self.maintainers = ['JO']
-        self.tags = {'production', 'external_resosurces'}
+        self.tags = {'production', 'external_resources'}
```

Comment:
We don't need the `external_resources` tag here, because we're not using the `resourcesdir`.